kubeflow/gcp/tests/prometheus_test.jsonnet

```diff
@@ -0,0 +1,229 @@
+    },
+    kind: "ConfigMap",
+    metadata: {
+      name: "prometheus",
+      namespace: "stackdriver",
+    },
+  }
+) &&
+
+std.assertEqual(
```

Comment:
Instead of doing std.assertEqual can we follow the pattern in
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/pytorch-job/tests/pytorch-job_test.jsonnet#L227

The problem with std.assertEqual is that it basically provides unintelligible errors when the test doesn't match (e.g. big blob string != other big blog string)

With the pattern in pytorch we output a json string with the results and test cases. The python test harness can then parse that and provide pretty formatted output to help debugging.

Reply:
that's much better. I'll convert all of the libsonnets to this form.

Reply:
Thanks.